cluster-autoscaler/simulator/node_info_utils.go

```diff
@@ -130,6 +145,9 @@ func podsExpectedOnFreshNode(sanitizedExampleNodeInfo *framework.NodeInfo, daemo
 		}
 	}
 	// Add all pending DS pods if force scheduling DS
+	// TODO(DRA): Figure out how to make this work for DS pods using DRA. Currently such pods would get force-added to the
+	// ClusterSnapshot, but the ResourceClaims reflecting their DRA usage on the Node wouldn't. So CA would be overestimating
+	// available DRA resources on the Node.
```

Comment:
This should be okay in practice for the "admin mode" monitoring use case because all devices on the new node would be available to pods, despite also having such a DS.